lib/pf/Switch/Dell/N1500.pm

```diff
@@ -199,9 +199,7 @@ sub getPhonesLLDPAtIfIndex {
                         && ($MACresult->{
                                 "$oid_lldpRemPortId.$cache_lldpRemTimeMark.$cache_lldpRemLocalPortNum.$cache_lldpRemIndex"
                             }
-                            =~ /([0-9A-Z]{2})\s?([0-9A-Z]{2})\s?([0-9A-Z]{2})\s?([0-9A-Z]{2})\s?([0-9A-Z]{2})\s?([0-9A-Z]{2}).*/i
-                        )
-                        )
+                            =~ /^(?:0x)?([0-9A-Z]{2})([0-9A-Z]{2})([0-9A-Z]{2})([0-9A-Z]{2})([0-9A-Z]{2})([0-9A-Z]{2})(?::..)?$/i))
```

Comment:
Just a detail this one will be better:

```
 =~ /^ (?:0x)?([0-9A-F]{2})([0-9A-F]{2})([0-9A-F]{2})([0-9A-F]{2})([0-9A-F]{2})([0-9A-F]{2})(?::..)?$/i))
```

Reply:
Hi @fdurand,

I replaced `A-Z` by `A-F` in regex.

I see a space at beginning of your regex, do you want me to add it or it's a typo ? If it's a typo, you can merge.